test/perf/micro/java/src/main/java/org/julialang/benchmarks/FibBenchmark.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2014, Oracle America, Inc.
```

Comment:
Mu bad, I reused class generated by maven jmh archetype and doesn't mention this header. Deleted.